elections/2020/governance-committee-election.md

```diff
@@ -1,5 +1,7 @@
 # OpenTelemetry 2020 Governance Committee election
 
+Please see [the list of candidates running in the election here](./governance-committee-candidates.md).
```

Comment:
Can you make this **bold**?

🤦 I'm so annoyed with myself for making this mistake!!